src/ray/common/task/task_spec.h

```diff
@@ -61,6 +61,10 @@ inline bool operator==(const ray::rpc::SchedulingStrategy &lhs,
             rhs.placement_group_scheduling_strategy()
                 .placement_group_capture_child_tasks());
   }
+  case ray::rpc::SchedulingStrategy::kNodeLabelSchedulingStrategy: {
+    return (lhs.node_label_scheduling_strategy().DebugString() ==
```

Comment:
Is it safe to compare the DebugString()?

Reply:
I also thought that `DebugString` was not quite good. I have now change to `SerializeAsString()`.